.dockerignore

```diff
@@ -4,3 +4,8 @@ tmp/
 .kube/
 bin/
 config/
+
+# Acceptance test suites don't need recompiling into our docker image
+cmd/acceptance/
```

Comment:
Ignoring these means you can modify the acceptance test, run `go run cmd/acceptance/main.go prepare`, and your docker image is already cached.